configs/default/HARC-HAKRCF405ICM.config

```
@@ -0,0 +1,129 @@
+# Betaflight / STM32F405 (S405) 4.2.4
+
+#define USE_GYRO_SPI_ICM42688P 
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name HAKRCF40ICM
```

```suggestion
board_name HAKRCF405ICM
```

Are you sure this is a good board name? If in future there are other gyros on board this name will be very misleading.